packages/flutter/test/painting/text_painter_test.dart

```diff
@@ -1740,3 +1788,19 @@ class MockCanvasWithDrawParagraph extends Fake implements Canvas {
     centerX = offset.dx + paragraph.width / 2;
   }
 }
+
+// Load the font used in the test.
+Future<void> _loadFont() async {
```

Comment:
@LongCatIsLooong how does this look for loading a different font from the test one? [Looks like this is what we recommend to do](https://github.com/flutter/flutter/blob/master/docs/contributing/testing/Flutter-Test-Fonts.md), but it feels flacky.